src/main/java/service/impl/CsvFileReaderServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package service.impl;
+
+import au.com.bytecode.opencsv.CSVReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import service.CsvFileReaderService;
+
+public class CsvFileReaderServiceImpl implements CsvFileReaderService {

```

Comment:
I guess it is possible that you will have any other file readers soon. And it would be cool if you could reuse one universal interface for them. Let's rename CsvFileReaserService to FileReaderService;